backend/de.metas.adempiere.adempiere/base/src/main/java/de/metas/organization/impl/OrgDAO.java

```diff
@@ -143,7 +143,8 @@ private OrgInfo retrieveOrgInfo(@NonNull final OrgId orgId, final String trxName
 		return toOrgInfo(record);
 	}
 
-	private I_AD_OrgInfo retrieveOrgInfoRecordOrNull(final OrgId adOrgId, final String trxName)
+	@Override
+	public I_AD_OrgInfo retrieveOrgInfoRecordOrNull(final OrgId adOrgId, final String trxName)
```

Comment:
instead of making this method public, pls introduce another method:

public OptionalInt getOrgImage(final OrgId orgId, final String imageColumnName)